crates/libs/implement/src/lib.rs

```diff
@@ -505,17 +526,27 @@ impl syn::parse::Parse for UseTree2 {
                 input.parse::<syn::Token![::]>()?;
                 Ok(UseTree2::Path(UsePath2 { ident, tree: Box::new(input.parse()?) }))
             } else if input.peek(syn::Token![=]) {
-                if ident != "TrustLevel" {
+                if ident == "TrustLevel" {
+                    input.parse::<syn::Token![=]>()?;
+                    let span = input.span();
+                    let value = input.call(syn::Ident::parse_any)?;
+                    match value.to_string().as_str() {
+                        "Partial" => Ok(UseTree2::TrustLevel(1)),
+                        "Full" => Ok(UseTree2::TrustLevel(2)),
+                        _ => Err(syn::parse::Error::new(span, "`TrustLevel` must be `Partial` or `Full`")),
+                    }
+                } else if ident == "ImplLocation" {
```

Comment:
Trying to think of the best way to phrase this to describe to the end-user what it will do.

Currently I'm thinking:
```
ImplInterfaceOn = OuterImpl
ImplInterfaceOn = InnerType
```